test/libsolidity/syntaxTests/license/license_bottom.sol

```diff
@@ -0,0 +1,3 @@
+// This test is actually useless, as the test suite adds the automatic preamble.
+contract C {}
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
I have added `*.sol` equivalents for the metadata tests, just like @axic originally did, but there are a lot of them and they are indeed useless, so maybe better to remove them after all?

Reply:
The fuzzer copies tests from the `test` directory. So perhaps retain them?

Reply:
OK. That sounds like a good enough reason to leave them in.